src/box/sql/func.c

```diff
@@ -522,6 +522,121 @@ func_trim_str(struct sql_context *ctx, int argc, const struct Mem *argv)
 		ctx->is_aborted = true;
 }
 
+static int
+sql_utf8_symbol_count(const char *str, int byte_len)
+{
+	int symbol_count = 0;
+	int str_end = 0;
+	UChar32 c = 0;
+	while (str_end < byte_len) {
+                U8_NEXT((uint8_t *)str, str_end, byte_len, c);
```

Comment:
Please use tabs instead of spaces (see [style guide](https://www.tarantool.io/en/doc/latest/dev_guide/c_style_guide/)). Here and below.